src/app/pages/backend-integration/diagram/backend-integration-diagram.component.scss

```diff
@@ -0,0 +1,108 @@
+@import '../../../@theme/styles/themes';
+
+@include nb-install-component() {
+  width: 100%;
+  height: 0;
+  position: relative;
+  padding-top: 80%; /* Aspect Ratio */
+  //background-color: transparentize(red, 0.8);
+  font-size: 0.9vw;
+
+  $subheader-color: nb-theme(text-hint-color);
+
+  .header, .subheader {
+    justify-content: center;
+  }
+
+  & > div {
+    position: absolute;
+    top: 0;
+    left: 0;
+    right: 0;
+    bottom: 0;
+    margin-bottom: 0;
+
+    & > .header {
+      font-size: 1.25em;
+      font-weight: bold;
+    }
+
+    & > .subheader {
+      font-size: 0.9em;
+      color: $subheader-color;
+    }
+  }
+
+  div {
+    display: flex;
+    text-align: center;
+  }
+
+  .json-container {
+    font-weight: bold;
+    div {
+      margin: 0 1em;
+    }
+  }
+
+  .headers-container {
+    height: 2.5rem;
+
+    & > .header {
+      font-size: 0.9em;
+      height: 0.9rem;
+      font-weight: bold;
+    }
+    & > .subheader {
+      font-size: 0.7em;
+      height: 0.7rem;
+      color: $subheader-color;
+    }
+  }
+
+  .dashboard-container {
+    width: 40%;
+  }
+
+  .json-container {
+    width: fit-content;
+
+    nb-icon {
+      width: 0.75em;
+      height: 0.75em;
+    }
+  }
+
+  .backend-container {
+    width: 40%;
+  }
+
+  .border-dashed {
+    border: 2px dashed nb-theme(border-basic-color-4);
+  }
+
+  .pad1 {
+    background-color: transparentize(nb-theme(background-basic-color-2), 0.5);
```

Comment:
Similar to fonts, could you try to pick something from basic transparent options? https://github.com/akveo/nebular/blob/9a9f127c762c6b5cdaee6337bdb16b40338ed9b6/src/framework/theme/styles/themes/_default.scss#L124